src/UI/Core/ListBox.cpp

```diff
@@ -198,8 +198,8 @@ void ListBox::dropAllItems()
 	_updateItemDisplay();
 }
 
-
-void ListBox::onMouseDown(EventHandler::MouseButton button, int x, int y)
+//TODO: Intended to only react if LMB?
```

Comment:
That's a very good point. Again, this should be an issue on GitHub, rather than a source code comment.

Related, there may be hotkeys, such as `<Enter>` which might function similarly.